### PR TITLE
Add support to retrieve the information stored in Datastore using a GET request to the JobsServlet

### PR DESCRIPTION
**Pull request by andreeanica16**
1. Adds class JobJSON that models the information sent as a JSON file back to the script file.
2. Add function getJobsFromDatastore() to the JobStoreCenter class that queries Datastore for the projectId specified in the request, fetches all the entities, format them as a JobJSON and sends all of them back to the caller functions in the javascript script file. This function retrieves all the jobs associated with a project (Running and Finalised).

### CODE REVIEW

**Comment by andreeanica16** [comment]
@iht @juliocc @tblanshard

File: src/main/webapp/config.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
  * @author tblanshard
```
Comment (iht) [inline src/main/webapp/config.js:5]: This file could be included in the .gitignore, so we don't disclose the project details.

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -10,7 +10,8 @@
     </head>
     <body onload="initBody();">
         <h2>Hello World!</h2>
-        <button class="button" onclick="updateJobDatabase()">Fetch Info</button>
+        <button class="button" onclick="updateProjectDatabase()">Fetch Info</button>
+        <button class="button" onclick="getJobsFromProject('bt-dataflow-sql-demo')">Show info</button>
```
Comment (iht) [inline src/main/webapp/index.html:14]: Hardcoded project id. I think this should be read from `config.js`

**Review by iht — APPROVED** [review]
LGTM

Two minor comments added. PTAL

File: src/main/webapp/config.js (hunk 1 above)
Comment (andreeanica16) [reply src/main/webapp/config.js:5]: I have't realised we should this file in .gitigonore. Thanks for your suggestion :D.

File: src/main/webapp/index.html (hunk 2 above)
Comment (andreeanica16) [reply src/main/webapp/index.html]: Changed this in https://github.com/googleinterns/step229-2020/pull/22/commits/3ba0a2ff46e7491182f0eb136bf26bd95bc3c4db
Thanks :D

File: src/main/webapp/config.js (hunk 1 above)
Comment (andreeanica16) [reply src/main/webapp/config.js]: Modified .gitignore here https://github.com/googleinterns/step229-2020/pull/22/commits/3ba0a2ff46e7491182f0eb136bf26bd95bc3c4db

File: src/main/webapp/script.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ function initBody() {
 }
 
 function checkPermissions() {
-  fetch('/check-permissions?projID='+config.projectID)
+  fetch('/check-permissions?projID='+configLogs.projectID)
```
Comment (tblanshard) [inline src/main/webapp/script.js:13]: Have you changed the name of the config json structure in config? Just so I can make the same change if needed?

Comment (andreeanica16) [reply src/main/webapp/script.js:13]: Yes. I tried to solve _Uncaught SyntaxError: Identifier 'config' has already been declared_ by changing the name.

**Pull request merged**